code_challenges/advent_of_code/supply_stacks.py

```diff
@@ -0,0 +1,59 @@
+def get_moves():
+
+    with open(
+        "/Users/amelia/projects/code-challenges/code_challenges/advent_of_code/supply_stacks_moves"
+    ) as move:
+        moves = move.read().split("\n")
+
+    return moves
+
+
+def crates():
+    crates = {
+        "1": ["F", "T", "C", "L", "R", "P", "G", "Q"],
+        "2": ["N", "Q", "H", "W", "R", "F", "S", "J"],
+        "3": ["F", "B", "H", "W", "P", "M", "Q"],
+        "4": ["V", "S", "T", "D", "F"],
+        "5": ["Q", "L", "D", "W", "V", "F", "Z"],
+        "6": ["Z", "C", "L", "S"],
+        "7": ["Z", "B", "M", "V", "D", "F"],
+        "8": ["T", "J", "B"],
+        "9": ["Q", "N", "B", "G", "L", "S", "P", "H"],
+    }
+    return crates
+
+
+def get_top_crates():
+    map_of_crates = crates()
+    moves = get_moves()
+    moves = [move.split(" ") for move in moves]
+    for move in moves:
+        for _ in range(int(move[1])):
+            # print(map_of_crates[move[5]])
+            map_of_crates[move[5]].append(map_of_crates[move[3]].pop())
```

Comment:
Any time you find yourself doing this `move[5], move[1], move[3]` instead consider doing
`for thing, _, anther_thing, _, the_last_thing in moves`